arch-secure-boot

```diff
@@ -12,8 +12,13 @@ ESP="${ESP:-/efi}"
 EFI="${EFI:-/EFI/arch}"
 KERNEL="${KERNEL:-linux}"
 NAME="${NAME:-secure-boot-$KERNEL}"
+SNAPPER_OPTIONS="${SNAPPER_OPTIONS:---no-dbus}"
 SUBVOLUME_ROOT="${SUBVOLUME_ROOT:-root}"
 SUBVOLUME_SNAPSHOT="${SUBVOLUME_SNAPSHOT:-snapshots/%1/snapshot}" # %1 is replaced with snapshot ID
+UCODE="${UCODE:-*}"
```

Comment:
> One more thing, after all that journey, I wanted to be able to only include Intel ucode because I don't have any AMD CPU here. So I also implemented this possibility in the script.

Does it mean that you have `amd-ucode` package installed, even though you don't have AMD processor? Otherwise wildcard would match only intel ucode anyway 🙂